backend/src/domain/mail/mailParser.js

```diff
@@ -92,14 +59,7 @@ const componerMailResumen = (reunion, temas, fecha) => {
           <% } else { %>  
             <p><%= tema.conclusion ? tema.conclusion : 'No hay minuta' %></p>
```

Comment:
```suggestion
            <p><%= tema.conclusion ?? 'No hay minuta' %></p>
```

no estoy seguro de ese cambio, o si hay que usar `||`...